### PR TITLE
kwallet-query: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by spageektti — APPROVED** [review]
Hello! Thanks for your contribution. Looks good to me!

<hr>

By the way, this is my 1000th pull request reviewed on GitHub.

File: pages/linux/kwallet-query.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# kwallet-query
+
+> Read and write to a KDE Wallet.
+> More information: <https://manned.org/kwallet-query>.
+
+- List all entries in the Passwords folder of `kdewallet`:
```
Comment (gutjuri) [inline pages/linux/kwallet-query.md:6]:
```suggestion
- List all entries in the passwords folder of `kdewallet`:
```

**Review by gutjuri — APPROVED** [review]

File: pages/linux/kwallet-query.md (hunk 1 above)
Comment (Managor) [reply pages/linux/kwallet-query.md:6]: No the folder is literally named "Passwords"

Comment (gutjuri) [reply pages/linux/kwallet-query.md:6]:
```suggestion
- List all entries in the `Passwords` folder of `kdewallet`:
```

Okay, then maybe like this?

**Pull request merged**